custom/icds_reports/management/commands/queue_person_cases.py

```diff
@@ -0,0 +1,60 @@
+from django.core.management.base import BaseCommand
+from corehq.apps.userreports.tasks import _get_config_by_id
+from corehq.apps.userreports.util import get_table_name
+from corehq.apps.userreports.models import AsyncIndicator
+from django.db import connections
+
+from dimagi.utils.chunked import chunked
+
+
+class Command(BaseCommand):
+    help = "Rebuild Bihar person cases (Child and AG Cases)"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'type_of_cases',
+            type=str
+        )
+
+    def handle(self, *args, **kwargs):
+        # by default execute child cases
+        type_of_cases = kwargs['type_of_cases'] if kwargs['type_of_cases'] else "child_cases"
+        person_config = _get_config_by_id('static-icds-cas-static-person_cases_v3')
+        # sort by supervisor_id and doc_id to improve the performance, sorting is needed to resume the queueing
+        # if it fails in between.
+        # AG Cases
+        query = ""
+        if type_of_cases == 'ag_cases':
+            # AG CASES
+            table_name = get_table_name('icds-cas', 'static-person_cases_v3')
+            # includes all the valid cases for the march april and may 2020
+            ag_end_range = '01-06-2009'
```

Comment:
wont the end date would be 2009-05-31?

Reply:
nope I am taking `<` not `<=`

Reply:
I was refering to the agg_Awc. where it is `<` the end of the month. But I think it's fine to have the last day  of the month also get included